internal/cmd/egctl/envoy_stats.go

```diff
@@ -0,0 +1,195 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package egctl
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"sync"
+
+	kube "github.com/envoyproxy/gateway/internal/kubernetes"
+	"github.com/spf13/cobra"
+	"github.com/tetratelabs/multierror"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	summaryOutput          = "short"
```

Comment:
istio `short` for consistency, let's use `statsd` or something like that. cc @arkodg , any suggestions?